### DISCUSSION TITLE
0.9.0 Can't disable performance mode; download won't start without the "dialogue" enabled

### DISCUSSION THREAD

**Discussion by Jimmisama** [robertodoering/harpy#660] [General]
1. Performance mode doesn't do anything in the new update. It looks like it's always enabled 

2. Download button won't work unless  "download dialogue" option enabled

After reinstalling or switching to the pro version, the bugs persist.

Also feature requests : replies counts too?

**Comment by robertodoering** [comment]
>Performance mode doesn't do anything in the new update. It looks like it's always enabled

The old card animation, which got disabled with the performance mode setting, has been removed. However the setting still has an effect on other animations in the app (e.g. drawer animation, image loading shimmer, etc.)

> Performance mode doesn't do anything in the new update. It looks like it's always enabled

Thanks! Will be fixed with the next update.

> Also feature requests : replies counts too?

Not yet possible, need to migrate to the new api first, which is planned

**Comment by Jimmisama** [comment]
Another feature request: avoiding repeated downloads
